@bencart Could you please address these issues - 
```
lib/ansible/modules/cloud/amazon/aws_acm_facts.py:0:0: E326 Argument 'statuses' in argument_spec defines choices as ([]) but documentation defines choices as ([['PENDING_VALIDATION'], ['ISSUED'], ['INACTIVE'], ['EXPIRED'], ['VALIDATION_TIMED_OUT']])
test/sanity/validate-modules/ignore.txt:3:1: A102 Remove since "lib/ansible/modules/cloud/amazon/aws_acm_facts.py" passes "E322" test
test/sanity/validate-modules/ignore.txt:4:1: A102 Remove since "lib/ansible/modules/cloud/amazon/aws_acm_facts.py" passes "E323" test
```

--- a/lib/ansible/modules/cloud/amazon/aws_acm_facts.py
+++ b/lib/ansible/modules/cloud/amazon/aws_acm_facts.py
@@ -21,8 +21,7 @@ options:
   statuses:
     description:
       - Status to filter the certificate results
-    choices: ['PENDING_VALIDATION', 'ISSUED', 'INACTIVE', 'EXPIRED', 'VALIDATION_TIMED_OUT']
-
+    choices: ['PENDING_VALIDATION', 'ISSUED', 'INACTIVE', 'EXPIRED', 'VALIDATION_TIMED_OUT', 'REVOKED', 'FAILED']
 requirements:
   - boto3
 author:
@@ -311,7 +310,7 @@ def main():
     argument_spec.update(
         dict(
             domain_name=dict(aliases=['name']),
-            statuses=dict(type='list'),
+            statuses=dict(type='list', choices=['PENDING_VALIDATION', 'ISSUED', 'INACTIVE', 'EXPIRED', 'VALIDATION_TIMED_OUT', 'REVOKED', 'FAILED']),
         )
     )
     module = AnsibleModule(argument_spec=argument_spec, supports_check_mode=True)

--- a/test/sanity/validate-modules/ignore.txt
+++ b/test/sanity/validate-modules/ignore.txt
@@ -1,7 +1,5 @@
 lib/ansible/modules/cloud/amazon/_ec2_ami_find.py E322
 lib/ansible/modules/cloud/amazon/_ec2_ami_find.py E323
-lib/ansible/modules/cloud/amazon/aws_acm_facts.py E322
-lib/ansible/modules/cloud/amazon/aws_acm_facts.py E323
 lib/ansible/modules/cloud/amazon/aws_api_gateway.py E322
 lib/ansible/modules/cloud/amazon/aws_application_scaling_policy.py E322
 lib/ansible/modules/cloud/amazon/aws_application_scaling_policy.py E326

